### DISCUSSION TITLE
Reproducing the screenshots

### DISCUSSION THREAD

**Discussion by ajrouvoet** [NeogitOrg/neogit#1217] [Q&A] [answered]
The screenshots in the README look gorgeous. What do I need to reproduce it? Are there themes that make neogit look this good out of the box? Is a theme sufficient or do I need other tweaks?

**Comment by CKolkey** [comment, marked as answer]
Thanks! You'll have to plunder my dotfiles.. But here are the high-level points:
- Theme: https://github.com/olimorris/onedarkpro.nvim
- Custom Highlight Group Definitions: https://github.com/CKolkey/config/blob/master/nvim/lua/ckolkey/config/ui/highlights.lua#L180-L190
- Modified colors: https://github.com/CKolkey/config/blob/master/nvim/lua/ckolkey/config/ui/colors.lua

**Reply by ajrouvoet** [reply]
Thank you kindly for the references!

If somewhat portable, perhaps the gist of it could be included in either the plugin or an example configuration in the repository.
Or as an optional shipped theme with neogit support.

**Reply by CKolkey** [reply]
Ehhh, I wouldn't consider it portable. I had fun a few years ago writing a UI layer in my config. I've upstreamed as much as was reasonable into this plugin - I don't wanna take everyone's fun away by doing _everything_ ;)

**Reply by ajrouvoet** [reply]
Also fair, of course!

**Comment by gldtn** [comment]
@CKolkey, 

Is there a highlight group for Floating/Popup windows/Border.. seems like you can only change the background color of "NeogitNormal" which also changes the Float. Also I'm trying to change the Float Border color as well.

I did find a list of possible highlights here: https://github.com/NeogitOrg/neogit/blob/master/lua/neogit/lib/hl.lua 

But couldn't find anything related to border as far as I could tell.

Thanks for sharing you config!

**Reply by CKolkey** [reply]
You can relink highlight groups with something like this in `~/.config/nvim/after/ftplugin/NeogitStatus.lua`

```lua
local win_id = vim.api.nvim_get_current_win()
local winhl = vim.api.nvim_win_get_option(win_id, "winhighlight")
vim.api.nvim_win_set_option(
  win_id,
  "winhighlight",
  winhl .. "Normal:NeogitNormal,WinSeparator:NeogitWinSeparator,CursorLineNr:NeogitCursorLineNr"
)
```

However, that's now standard in neogit `nightly` branch.

**Comment by gldtn** [comment]
Did the commit options window(when u hit `c` the split window) change? I use to be able to highlight it with `NeogitWinSeparator`, but now it only shows a thick bar like a tabline.

**Reply by CKolkey** [reply]
Yes, I recently made it a floating window to have more control over how it's presented. Configured here: https://github.com/NeogitOrg/neogit/blob/b81f37ec3169153394084cd564d7dc5871f0729a/lua/neogit/lib/buffer.lua#L336-L352

If there is some aspect you'd like exposed for user config, lemme know.

**Reply by gldtn** [reply]
Ahh, I see.. I'm assuming I can't really target as is currently with highlights. Would be nice to be able to target border/borderchar colors(fg/bg). I see you have title commented out, if it ends up being an option would be nice to target that as well.

Also, I notice there is no highlight for this particular popup background, unless I missed it. Would be awesome to be able to set different background for these style of popups.

As always, thank you!
